packages/cli/src/commands/alias/ls.ts

```diff
@@ -51,7 +51,7 @@ export default async function ls(
     ...paginationOptions
   );
   output.log(`aliases found under ${chalk.bold(contextName)} ${lsStamp()}`);
-  output.log(printAliasTable(aliases));
+  client.stdout.write(printAliasTable(aliases));
```

Comment:
I think this is missing a `\n`

Have you considered using `console.log()` instead?

Reply:
Agree, we probably want to add a `\n`. However we don't want to use `console.log()` so that we can assert in the tests.

Reply:
In my tests it has a newline at the end, just not two newlines. Happy to add in `\n` but I'm not sure if we need two newlines?

Existing:

```bash
➜  vercel git:(andymckay/back-to-stdout) ✗ cat -e stdout.txt
  source                                                     url                                                        age$
...[snip]
  15-binary-handler-bbp1gxgo8-andymckay.vercel.app           15-binary-handler.vercel.app                                6d$
  14-unicode-handler-f7eg54s4k-andymckay.vercel.app          14-unicode-handler-andymckay.vercel.app                     6d$
$
$
```

New:

```bash
➜  vercel git:(andymckay/back-to-stdout) ✗ cat -e stdout-dev.txt
  source                                                     url                                                        age$
...[snip]
  15-binary-handler-bbp1gxgo8-andymckay.vercel.app           15-binary-handler.vercel.app                                6d$
  14-unicode-handler-f7eg54s4k-andymckay.vercel.app          14-unicode-handler-andymckay.vercel.app                     6d$
$
```

Reply:
There's no need for a `\n` since `printAliasTable()` already appends and returns `\n\n`: https://github.com/vercel/vercel/pull/9227/files#diff-f60b8447303c4253e468e88965c478eaee1a8c04feaa48e371c98ceb78d87c27R86.